### PR TITLE
Added styled notes on all opcodes

### PR DESCRIPTION
**Pull request by hjred**
Added styled notes on the first 100 opcodes for all games. Changed PST "does not effect the protagonist" if found and changed "Do not use" IWD2 effects to empty ones.

Added info and important include files.

### CODE REVIEW

File: _opcodes/op012-bg2.html [hunk 1]
```diff
@@ -72,8 +72,33 @@
...
+If 'Damage Amount' and the Dice values are both non-zero, the total damage inflicted is: 'Damage Amount' + Dice value.
+{% endcapture %}
+
+{% include info.html %}
+
+{% capture note %}
+'Damage Amount' can be positive or negative (though the damage sound will play in either case).
+{% endcapture %}
+
+{% include info.html %}
```
Comment (lynxlynxlynx) [inline _opcodes/op012-bg2.html]: How does this look in the result? It seems it would be quite spammy. I'd just move the dice stuff into a paragraph, part of the description.

File: _opcodes/op006.html [hunk 2]
```diff
@@ -15,7 +15,13 @@
...
 <br/>1   Flat Value Modifier -> Charisma = 'Statistic Modifier' value
 <br/>2   Percentage Modifier -> Charisma = (Charisma * 'Statistic Modifier' value) / 100
-<br/>'Statistic Modifier' can be positive or negative.
+
+{% capture note %}
+'Statistic Modifier' can be positive or negative.
+{% endcapture %}
+
+{% include info.html %}
+
```
Comment (lynxlynxlynx) [inline _opcodes/op006.html]: These seem excessive. While it is true that the statement does not hold for all opcodes, I don't think it is so important to be displayed so prominently.

File: _opcodes/op017-bg2.html [hunk 3]
```diff
@@ -15,10 +15,26 @@
...
+
+{% include info.html %}
+
+<br/>Known values for the upper word of 'Type' are:
 <br/>1 Cure death
 <br/>2 Removes all non timing-mode-9 effects
 <br/>3 Cure death and remove non timing-mode-9 effects
+
+{% capture note %}
+The upper word of the 'Type' field can further control the style.
```
Comment (lynxlynxlynx) [inline _opcodes/op017-bg2.html]: This belongs to the description.

File: _opcodes/op020-bg1.html [hunk 4]
```diff
@@ -14,5 +14,15 @@
...
-<br/>Improved Invisibility lingers after the duration is spent and works very permanent if used as a While Equipped effect.
+
+{% capture note %}
+Improved Invisibility does not provide saving throw bonuses.
+{% endcapture %}
+
+{% include note.html %}
+
+{% capture note %}
+Improved Invisibility lingers after the duration is spent and works very permanent if used as a While Equipped effect.
```
Comment (lynxlynxlynx) [inline _opcodes/op020-bg1.html]: chance for a typo fix.

File: _opcodes/op020-iwd1.html [hunk 5]
```diff
@@ -15,5 +15,15 @@
...
-<br/>Improved Invisibility lingers after the duration is spent and works very permanent if used as a While Equipped effect.
+
+{% capture note %}
+Improved Invisibility does not provide saving throw bonuses.
+{% endcapture %}
+
+{% include note.html %}
+
+{% capture note %}
+Improved Invisibility lingers after the duration is spent and works very permanent if used as a While Equipped effect.
```
Comment (lynxlynxlynx) [inline _opcodes/op020-iwd1.html]: same here

File: _opcodes/op020-pst.html [hunk 6]
```diff
@@ -11,4 +11,9 @@
 pst: 1
 ---
 Leave both parameters at all zeros for this Effect to work.
-<br/>This effect will not work with a timing mode of Duration or Permanent. All other timing modes function correctly.
+
+{% capture note %}
+This effect will not work with a timing mode of Duration or Permanent. All other timing modes function correctly.
```
Comment (lynxlynxlynx) [inline _opcodes/op020-pst.html]: Sounds like a warning would be better.

File: _opcodes/op021-iwd2.html [hunk 7]
```diff
@@ -15,5 +15,15 @@
...
-<br/>NB. This effect does not work with Timing Mode 1.
+
+{% capture note %}
+'Statistic Modifier' can be positive or negative.
+{% endcapture %}
+
+{% include info.html %}
+
+{% capture note %}
+This effect does not work with Timing Mode 1.
```
Comment (lynxlynxlynx) [inline _opcodes/op021-iwd2.html]: warning?

File: _opcodes/op023-bg2.html [hunk 8]
```diff
@@ -11,4 +11,9 @@
 pst: 0
 ---
 Sets the Morale stat of the targetted creature(s) to 10.
-<br/>NB: this is a gimped effect in bg2, unlike the rest of the games!
+
+{% capture note %}
+this is a gimped effect in bg2, unlike the rest of the games!
+{% endcapture %}
```
Comment (lynxlynxlynx) [inline _opcodes/op023-bg2.html]: meaning it doesn't work, so maybe a warning instead.

File: _opcodes/op066-iwd1.html [hunk 9]
```diff
@@ -19,4 +19,9 @@
 <br/>0 Default
 <br/>1 Fade In
 <br/>2 Fade Out
-<br/>This effect sets stat #60 (TRANSLUCENT) to param #1.
+
+{% capture note %}
+This effect sets stat #60 (TRANSLUCENT) to param #1.
+{% endcapture %}
```
Comment (lynxlynxlynx) [inline _opcodes/op066-iwd1.html]: these should just remain as is. There's nothing special or extra about the stat being set to warrant more attention from the viewer.

File: _opcodes/op076-iwd2.html [hunk 10]
```diff
@@ -11,5 +11,15 @@
...
-<br/>NB. A creatures scripts do not while the creature is feebleminded.
+
+{% capture note %}
+This effect ignores durations attributed to it.
+{% endcapture %}
+
+{% include note.html %}
+
+{% capture note %}
+A creatures scripts do not while the creature is feebleminded.
```
Comment (lynxlynxlynx) [inline _opcodes/op076-iwd2.html]: spelling fix potential — apostrophe and "run".

File: _opcodes/op078-bg1.html [hunk 11]
```diff
@@ -22,5 +22,15 @@
 <br/>7   Intelligence Penalty
 <br/>8   Wisdom Penalty
 <br/>9   Charisma Penalty
-<br/>'Damage' can be positive (penalty) or negative (bonus).
-<br/>Permanent timing modes require a duration for this effect to work.
+
+{% capture note %}
+'Damage' can be positive (penalty) or negative (bonus).
+{% endcapture %}
```
Comment (lynxlynxlynx) [inline _opcodes/op078-bg1.html]: This can be in the main paragraph.

File: _opcodes/op084-iwd2.html [hunk 12]
```diff
@@ -1,17 +1,13 @@
...
 iwd1: 0
 iwd2: 1
 pst: 0
 ---
-*DO NOT USE*
-<br/>Applies the modifier value specified by the 'Statistic Modifier' field in the style specified by the 'Type' field.
-<br/>Known values for 'Type' are:
-<br/>0   Cumulative Modifier -> Damage = Damage + 'Statistic Modifier' value
-<br/>1   Flat Value Modifier -> Damage = 'Statistic Modifier' value
+This opcode has no effect.
```
Comment (lynxlynxlynx) [inline _opcodes/op084-iwd2.html]: Do we know this? Maybe it crashes.

Comment (hjred) [inline _opcodes/op084-iwd2.html]: I don't know this yet. Will revert the changes for all *do not use* iwd2 opcodes

File: _opcodes/op078-bg1.html (hunk 11 above)
Comment (hjred) [inline _opcodes/op078-bg1.html]: Sometimes this is used with a note and sometimes not, so i changed all of them to be consistent across all opcodes

File: _opcodes/op076-iwd2.html (hunk 10 above)
Comment (hjred) [inline _opcodes/op076-iwd2.html]: Yeah, will change this

File: _opcodes/op066-iwd1.html (hunk 9 above)
Comment (hjred) [inline _opcodes/op066-iwd1.html]: Ok, will change back

File: _opcodes/op023-bg2.html (hunk 8 above)
Comment (hjred) [inline _opcodes/op023-bg2.html]: Ok, will change to warning

File: _opcodes/op021-iwd2.html (hunk 7 above)
Comment (hjred) [inline _opcodes/op021-iwd2.html]: Will change

File: _opcodes/op020-pst.html (hunk 6 above)
Comment (hjred) [inline _opcodes/op020-pst.html]: Yeah, will change

File: _opcodes/op012-bg2.html (hunk 1 above)
Comment (hjred) [inline _opcodes/op012-bg2.html]: I don't know how it will look, i don't have the possibility to compile the templates. But i will look into it.

**Comment by hjred** [comment]
Will change noted things and apply it to the rest of the opcodes. They are temporarily completed.

**Comment by hjred** [comment]
All changes to the first 100 opcodes are made and these are applied to the rest. The number of notes were reduced were possible.

**Comment by hjred** [comment]
Finally i can preview the site on my machine and so far everything looks good :)

**Comment by lynxlynxlynx** [comment]
These new commits have no place in this pull request. Please open separate ones for those that don't have it yet and force push just the opcode stuff here. Let me know if you need help with git.

**Comment by hjred** [comment]
The problem is i can't open another pull request, when one is already open (exception: from different branches). People on github suggest cherry picking commits in this case.

**Comment by lynxlynxlynx** [comment]
yes, just move them to new branches*. In one project I'm involved with direct commits to master are even disabled and everything has to come from topic branches.

* either via cherrypicking or rebase -i

File: _opcodes/op020-bg1.html [hunk 13]
```diff
@@ -16,13 +16,13 @@
...
 {% capture note %}
-Improved Invisibility does not provide saving throw bonuses.
+'Improved Invisibility' does not provide saving throw bonuses.
 {% endcapture %}
 
 {% include note.html %}
 
 {% capture note %}
-Improved Invisibility lingers after the duration is spent and works very permanent if used as a While Equipped effect.
+'Improved Invisibility' lingers after the duration is spent and works permanent if used as a 'While Equipped' effect.
```
Comment (lynxlynxlynx) [inline _opcodes/op020-bg1.html]: permanently.

File: _opcodes/op020.html [hunk 14]
```diff
@@ -16,13 +16,13 @@
...
 {% capture note %}
-Improved Invisibility does not provide saving throw bonuses
+'Improved Invisibility' does not provide saving throw bonuses
 {% endcapture %}
 
 {% include note.html %}
 
 {% capture note %}
-Improved Invisibility lingers after the duration is spent and act as permanent if used as a While Equipped effect
+'Improved Invisibility' lingers after the duration is spent and act as permanent if used as a While Equipped effect
```
Comment (lynxlynxlynx) [inline _opcodes/op020.html]: ... or use this form in the others too.

**Comment by hjred** [comment]
Removed the other commits, only opcode stuff left.

File: _opcodes/op020-bg1.html (hunk 13 above)
Comment (hjred) [inline _opcodes/op020-bg1.html]: Oh ok :D

File: _opcodes/op116.html [hunk 15]
```diff
@@ -10,6 +10,16 @@
 iwd2: 1
 pst: 1
 ---
-Removes invisibility from the targeted creature(s).<br/>
-This effect ignores durations attributed to it.<br/>
+Removes invisibility from the targeted creature(s).
+
+{% capture note %}
+This effect ignores durations attributed to it.
```
Comment (lynxlynxlynx) [inline _opcodes/op116.html]: This is also something not worth making a fuss about. It's a removal opcode, not an immunity. A one shot — I'm surprised people deemed it worthy to mention this at all.

File: _opcodes/op453-iwd2.html [hunk 16]
```diff
@@ -10,4 +10,10 @@
 iwd2: 1
 pst: 0
 ---
-Disables the usage of Expertise, detects the activation/revocation of Power Attack, prints appropriate messages. This effect is used internally.
+Disables the usage of Expertise, detects the activation/revocation of Power Attack, prints appropriate messages.
+
+{% capture note %}
+This effect is used internally.
```
Comment (lynxlynxlynx) [inline _opcodes/op453-iwd2.html]: Many are, not noteworthy enough. It just has a connotation that it's use might be different or difficult. Same for the other feat spells.

File: _opcodes/op116.html (hunk 15 above)
Comment (lynxlynxlynx) [inline _opcodes/op116.html]: Same for the others, of course.

File: _opcodes/op453-iwd2.html (hunk 16 above)
Comment (hjred) [inline _opcodes/op453-iwd2.html]: My thought was, if somebody want's to use these, they should clearly see, that these are internal to the engine and maybe it's better to look for an alternative

File: _opcodes/op116.html (hunk 15 above)
Comment (hjred) [inline _opcodes/op116.html]: I added these to prevent programming errors, to avoid a trial and error programming style, setting pseudo-random values until the function does, what they want. The next one who uses their work maybe think they have to set it

Comment (lynxlynxlynx) [inline _opcodes/op116.html]: Nobody writes effects from scratch, it's much easier to ship the file or modify whatever it is attached to. A duration is always set, since it's an non-optional field and the point of the note is that it doesn't matter at all.

File: _opcodes/op453-iwd2.html (hunk 16 above)
Comment (lynxlynxlynx) [inline _opcodes/op453-iwd2.html]: The note says just that the effect is used by the engine, nothing more. The engine also uses the damage opcode and others, but they don't have this note. It needs more research whether these modal feat effects are any special at all. That's why it's pointless to stress the fact.

File: _opcodes/op116.html (hunk 15 above)
Comment (hjred) [inline _opcodes/op116.html]: Ok, so reverting to the old style? Or maybe changing to "The value should always be set to ..."?

File: _opcodes/op453-iwd2.html (hunk 16 above)
Comment (hjred) [inline _opcodes/op453-iwd2.html]: Ok, then will revert them.

File: _opcodes/op116.html (hunk 15 above)
Comment (lynxlynxlynx) [inline _opcodes/op116.html]: Old style, since the value (and likely type) doesn't matter.

Comment (hjred) [inline _opcodes/op116.html]: Changed

**Comment by lynxlynxlynx** [comment]
Thanks! I know it can be a very drudgy process. Even with the help of scripts, my eyes almost bled in the initial deduplication effort.

**Comment by hjred** [comment]
Yeah, without doubt, but great work ;) i looked into the action stuff and to rip them apart per action instead of per game seems ... adventurous
